specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2023-12-10-preview/cloudhsm.json

```diff
@@ -0,0 +1,967 @@
+          "type": "array",
+          "description": "List of private endpoint connection resources",
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "statusMessage": {
+          "type": "string",
+          "description": "Cloud HSM Cluster status message",
+          "readOnly": true
+        },
+        "cloudHsmClusterRestoreProperties": {
+          "$ref": "#/definitions/CloudHsmClusterRestoreProperties"
+        }
+      },
+      "description": "Properties of a Cloud HSM Cluster.",
+      "type": "object"
+    },
+    "CloudHsmClusterPatchParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "The Cloud HSM Cluster's tags",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The Cloud HSM Cluster's tag's additional properties"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/CloudHsmClusterSku",
+          "description": "SKU details"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity"
+        }
+      },
+      "description": "Patchable properties of the Cloud HSM Cluster",
+      "type": "object"
+    },
+    "CloudHsmClusterSecurityDomainProperties": {
+      "type": "object",
+      "description": "Security domain properties information for Cloud HSM cluster",
+      "properties": {
+        "fipsState": {
+          "format": "int32",
+          "type": "integer",
+          "description": "FIPS state information for security domain"
+        },
+        "activationStatus": {
+          "type": "string",
+          "description": "status of security domain activation"
+        }
+      }
+    },
+    "CloudHsmClusterSku": {
+      "type": "object",
+      "description": "Cloud Hsm Cluster SKU information",
+      "required": [
+        "name",
+        "family"
+      ],
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "Sku family of the Cloud HSM Cluster",
+          "enum": [
+            "B"
+          ],
+          "x-ms-client-default": "B",
+          "x-ms-enum": {
+            "name": "CloudHsmClusterSkuFamily",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "Sku name of the Cloud HSM Cluster",
+          "enum": [
+            "Standard_B1",
+            "Standard B10"
+          ],
+          "x-ms-enum": {
+            "name": "CloudHsmClusterSkuName",
+            "modelAsString": false
+          }
+        },
+        "capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Sku capacity"
+        }
+      }
+    },
+    "CloudHsmClusterBackupProperties": {
```

Comment:
Where is this referenced ? Shouldn't this be referenced under CloudHsmClusterProperties and CloudHsmClusterPatchProperties, to become part of PUT and PATCH Call?

Reply:
yes, thanks @sumantshiv I've missed this :)